hdr/debug.h

```diff
@@ -49,7 +49,7 @@
 #ifdef DEBUG_PRINT_COMPORT
 #define DebugPrintf(x) dbgc_printf x
 #else
-#define DebugPrintf(x) _printf x
+#define DebugPrintf(x) fdprintf x
```

Comment:
It should be done as a new backend,
like `DEBUG_PRINT_DOSEMU`.
But as it actually goes to dosemu anyway,
what problem does this solve?